examples/gitlab-centos7-atomicapp/README.md

```diff
@@ -0,0 +1,157 @@
+This is an atomic application based on the nulecule specification. Docker and Kubernetes are the supported providers. You'll need to run this from a workstation that has the atomic command and kubectl client that can connect to a kubernetes master.
+
+It is 3 a tier application based on redis, postgresql and gitlab. This is also an example Nulecule having multiple tier artifacts rather using graphs.
+
+### Option 1: Interactive
+
+Run the image. It will automatically use kubernetes as the orchestration provider.  It will prompt for all parameters in the Nulecule file that do not have default values:
+
+    $ [sudo] atomic run projectatomic/gitlab-centos7-atomicapp
+
+## Option 2: Unattended
```

Comment:
IMO _unattended_ should consist of 

```
atomic install ...
mv answers.conf.sample answers.conf
vi answers.conf
atomic run ...
```

We shouldn't suggest any copy&pasting if it's not necessary - get rid of copy&pasting stuff from github is exactly one of the reasons why Nulecule has been created.